python/lib/dcoscli/dcoscli/job/main.py

```diff
@@ -228,6 +233,40 @@ def _remove(job_id, stop_current_job_runs=False):
     return 0
 
 
+def _queue(job_id, json_flag=False, quiet=False):
+    """
+    :param job_id: Id of the job
+    :type job_id: str
+    :param json_flag: Output json if True
+    :type json_flag: bool
+    :param quiet: Output only job run ids if true
+    :type quiet: bool
+    :returns: process return code
+    :rtype: int
+    """
+
+    client = metronome.create_client()
+    deployment_list = client.get_queued_job_runs(job_id)
+
+    if not deployment_list and not json_flag:
+        msg = "There are no deployments in the queue"
```

Comment:
I wouldn't print a message to stdout here, empty output / table is fine. There is also the case where the quiet option is passed and the message might be treated as an ID.

Reply:
yeah... this is consistent with other commands... but I'm with you... I prefer things to be silent if successful